### PR TITLE
Quantum computing script updates

### LINKED ISSUE #765: Quantum Computing script to generate random numbers
**This template is strictly for requesting New Scripts, if you want a new feature (automating this Repo,**
**a change in structure) in the Repo,** 
**or you want to report a Bug. There are other templates which you should follow**

# Description
The script will use Quantum gates to create random numbers.

## Language

- [ ] Bash
- [ ] Go
- [ ] Javascript
- [ ] Powershell
- [x] Python
- [ ] Other (Please specify)

## Checklist:

- **Details on the Script**

| name            | about                                                  | labels  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| Quant_random | Generate random numbers using Quantum computing | feature | self-assign |

**Is your feature request related to a problem? Please describe.**: Quantum computing can be tough to understand unless you understand basic gates used in Quantum computation. This script will serve as an example for gate manipulation in Quantum Computing.

### PR DESCRIPTION
**Pull request by RutvikJ77**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

Fixes #765 

Replace `765` with the issue number which is fixed in this PR

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/Quantum_random/.env.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+API_KEY="XXXXXXXXXXXXX"
+
+// Here you have to save your own Private Access Token which will be used in Main code to fetch the data.
```
Comment (HarshCasper) [inline Python/Quantum_random/.env.example:3]: Remove this line

File: Python/Quantum_random/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+### How to use?
+
+- Intially register at [IBM quantum computing](https://quantum-computing.ibm.com/) go to my account section located in top-right where you will find your credentials and information on how to run Qiskit package locally.
+
+- Additionally, I have included a section in comments for you to save your account. I recommend to setup a virtual environment for it, so you can easily uninstall packages if you do not want to use them further.
+
+![Qiskit API](https://i.imgur.com/xJ3RVX5.png)
+
+### Just type in:
```
Comment (HarshCasper) [inline Python/Quantum_random/README.md:12]: Replace `Just type in` to `Launch the Script`

File: Python/Quantum_random/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Quantum random number generation
+This python script explains how classical computations can be done using Quantum gates. It generates a random number using Quantum computation.
+
+### How to use?
+
+- Intially register at [IBM quantum computing](https://quantum-computing.ibm.com/) go to my account section located in top-right where you will find your credentials and information on how to run Qiskit package locally.
```
Comment (HarshCasper) [inline Python/Quantum_random/README.md:6]: Put `go to my account section` as a second point

File: Python/Quantum_random/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+`python Quant_random.py`
+
+### Requirements
+In case you want to setup a virtual environment. Read [this](https://realpython.com/python-virtual-environments-a-primer/).
+
+Before running the script, just install qiskit for python using this command:
+
+`pip install qiskit`
+
```
Comment (HarshCasper) [inline Python/Quantum_random/README.md:22]: Can a snap be added on what does the script yields?

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Tiny changes. Looks good to merge!

File: Python/Quantum_random/README.md (hunk 4 above)
Comment (RutvikJ77) [reply Python/Quantum_random/README.md:22]: Sure!

**Review by HarshCasper — APPROVED** [review]

**Comment by HarshCasper** [comment]
Thanks! Your name would be visible on the Leaderboard now 😉

**Comment by RutvikJ77** [comment]
Thank You! One more thing I had not been allotted the previous 100 points for my first script and the additional leaderboard 25 points, though it's okay my aim was to contribute as much as I could and I am soo glad to be guided by you all. Thanks! Open source is amazing.

**Comment by HarshCasper** [comment]
> Thank You! One more thing I had not been allotted the previous 100 points for my first script and the additional leaderboard 25 points, though it's okay my aim was to contribute as much as I could and I am soo glad to be guided by you all. Thanks! Open source is amazing.

Can you quickly send me the link? I will try to get it through as quick as possible.

**Comment by RutvikJ77** [comment]
> > Thank You! One more thing I had not been allotted the previous 100 points for my first script and the additional leaderboard 25 points, though it's okay my aim was to contribute as much as I could and I am soo glad to be guided by you all. Thanks! Open source is amazing.
> 
> Can you quickly send me the link? I will try to get it through as quick as possible.

Thank you it was done. Short of 25 points but that's totally cool!. Here is the link https://github.com/HarshCasper/Rotten-Scripts/pull/746

**Pull request merged**